presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/WindowMatcher.java

```diff
@@ -66,12 +66,6 @@ public boolean matches(PlanNode node, Session session, Metadata metadata, Expres
         return actualCopy.isEmpty();
     }
 
-    private static boolean sameCall(FunctionCall left, FunctionCall right)
```

Comment:
this PR already depends on 3 other PRs, I do not want to add another dependency for such minor change